src/module.c

```diff
@@ -1583,15 +1583,15 @@ int RM_SetCommandACLCategories(RedisModuleCommand *command, const char *aclflags
  *
  * All fields except `version` are optional. Explanation of the fields:
  *
- * - `version`: This field enables compatibility with different Redis versions.
+ * - `version`: This field enables compatibility with different server versions.
  *   Always set this field to REDISMODULE_COMMAND_INFO_VERSION.
```

Comment:
This is a tricky one. We'll have to track both redis version compatibility and valkey version compatibility and we're talking about introducing a ValkeyModule API... TBD

Reply:
So what are you thinking - should we leave the comment as is - since we need a bigger decision about what this field means? Sounds like it is possible we will leave this "version" to be used for "Redis version" (i.e. locked at 7.2.4)

Reply:
Actually no problem. This macro only defines the version of the struct defined in the same header file.